packages/components/src/experience-control/README.md

```diff
@@ -0,0 +1,84 @@
+# Experience Control
```

Comment:
Can we omit manual READMEs for now? All the prop descriptions should live as TSDocs on the prop types,  and the main description can live like this:

```tsx
/**
 * A flexible component for ...
 *
 * ```jsx
 * import { ExperienceControl } from '@automattic/components';
 * (...rest of code snippet...)
 * ```
 */
export function MyComponent = {
```

This way the data is available in IntelliSense, and can be reused by docgens, like in Storybook too.

Related: https://github.com/Automattic/wp-calypso/pull/102827#discussion_r2065027286

Reply:
Agree. Fixed it here: https://github.com/Automattic/wp-calypso/pull/103000